packages/css/src/plugins/__tests__/sort-shorthand-declarations.test.ts

```diff
@@ -386,13 +386,61 @@ describe('sort shorthand vs. longhand declarations', () => {
         border-block-start: 1px solid;
         border-block-start-color: transparent;
       }
-      .b { all: unset; }
       .c { all: unset; }
+      .b { all: unset; }
       .d { border: none; }
       .f { border-block-start-color: transparent; }"
     `);
   });
 
+  it('sorts a subset of atomic classes taken from a real stylesheet', () => {
```

Comment:
this is a small subset of the jira stylesheet that i found was not being sorted correctly

Reply:
Got it, some other thoughts if they're _not_ tested in here should be `border > border-top > border-top-color` right?

Input:
```tsx
._abcd1234 { border-top-color: red }
._abcd1234 { border-top: 1px solid }
._abcd1234 { border: none }
._abcd1234:hover { border-top-color: red }
._abcd1234:hover { border-top: 1px solid }
._abcd1234:hover { border: none }
@media (…) { ._abcd1234 { border-top-color: red } }
@media (…) { ._abcd1234 { border-top: 1px solid } }
@media (…) { ._abcd1234 { border: none } }
```
Output:
```tsx
._abcd1234 { border: none }
._abcd1234 { border-top: 1px solid }
._abcd1234 { border-top-color: red }
._abcd1234:hover { border: none }
._abcd1234:hover { border-top: 1px solid }
._abcd1234:hover { border-top-color: red }
@media (…) { ._abcd1234 { border: none } }
@media (…) { ._abcd1234 { border-top: 1px solid } }
@media (…) { ._abcd1234 { border-top-color: red } }
```

The main question I'd have is do we still maintain this pseudo and media query sorting, the rest seems fine to me…